nav2_bringup/bringup/README.md

```diff
@@ -1,21 +1,21 @@
 # nav2_bringup
 
-The `nav2_bringup` package is an example bringup system for navigation2 applications.
+The `nav2_bringup` package is an example bringup system for Navigation2 applications.
 
-Notes: (June 2019, Dashing Release)
-* We recommend doing this on a Ubuntu 18.04 installation. We have build issues on 16.04. (see https://github.com/ros-planning/navigation2/issues/353)
-* This stack and ROS2 are still in heavy development and there are some bugs and stability issues being worked on, so please do not try this on a robot without taking *heavy* safety precautions. THE ROBOT MAY CRASH!
-* It is recommended to start with simulation using Gazebo before proceeding to run on a physical robot
-
-Install and build our code by following this guide:
-https://github.com/ros-planning/navigation2/blob/master/doc/BUILD.md
+### Pre-requisites:
+* [Install ROS 2](https://index.ros.org/doc/ros2/Installation/Dashing/)
+* [Install Navigation 2](https://github.com/ros-planning/navigation2/blob/master/doc/BUILD.md)
+* Install your robot specific package (ex:[Turtlebot 3](http://emanual.robotis.com/docs/en/platform/turtlebot3/ros2/))
```

Comment:
I think that's implicit, but good to be complete

Reply:
I like complete documentation especially if it is a single file tutorial. Once we have more categorized documentation, this can be removed. In my opinion, the tutorials should be in some kind of an order such as;

Beginner
1- Build/Install Navigation2
2- Bringup Navigation2
3- Robot Specific Tutorials...

Intermediate
5- Navigation2 with SLAM
6- Navigation2 using Voxel Layer
7- Navigation2 with Multiple Robots
8- Navigation2 and RL

Advanced
9- Navigation2 Parameter Tuning
.
-etc (I just made this up, nothing official)

so at the beginning of each tutorial we can say "this tutorial assumes that you have completed the previous tutorials" <linked to the previous tutorial> something similar to ros1 tutorials.

Reply:
I would challenge a few of those. In particular...

Beginner: 3 should just be removed entirely. The only community supported one is TB3, otherwise there's no value in more than 1 specific example. More utility would come from 9 that should really be in beginner: `how do I use this thing I just installed`. 

RL should be dropped, it doesnt work right now and it's going to be a long roadmap before that's ready for prime time. 6 on voxel is just normal, that's just a layer in the costmap and doesn't need its own special instruction. Actually if you read it, you'll find its 90% just obstacle_layer code with a few overloads for the voxel grid. 

7 is useful to very few people, I'd consider that to be an advanced topic,

Reply:
Semantics for this PR but since you brought it up

Reply:
Ah this preview also shows another line, what  this was talking about was the `robot specific packages` line in my initial comment. One would think if you're at a point that your trying to use navigation, you already have a robot, or you are implicitly using TB3 as a demo.

Reply:
It is just an example list. I didn't put too much thought into it. 

> Beginner: 3 should just be removed entirely. The only community supported one is TB3, otherwise, there's no value in more than 1 specific example.

I don't agree with that. navigation 1 has a lot of robot specific tutorials in their documentation. We don't have to copy what nav1 has, but I think it is valuable to show people that Navigation2 supports/works with a variety of robots. On the other hand, maintaining tutorials for many robots can be hard, and they will have a lot of repetitions.

I will definitely create a Navigation2 with TB3 tutorial when I get a chance.

Reply:
Other tutorials have just been user created, or company created as part of trying to sell their robots. The navigation1 stack never really even supported turtlebot, that was another group of people running the turtlebot 1/2

Reply:
this is here nor there for this PR